astro/src/content/docs/apis/_refresh-token-response-body-base.mdx

```diff
@@ -8,7 +8,7 @@ import JSON from 'src/components/JSON.astro';
     An array of Refresh Tokens
   </APIField>
   <APIField name={ props.base_field_name + ".applicationId" } type="String">
-    The Id of the Application for which this token is authorized to be exchanged for a new Access Token.
+    The Id of the Application for which this token is authorized to be exchanged for a new Access Token. If this Id is not present, the token is for internal use and should be ignored.
```

Comment:
I don't know that we want people to ignore this. It is a legit token - it represents an SSO session. Someone could use this to revoke the SSO session, etc. We use this API in our admin UI to show sessions - one of which is the SSO token.

The only limitation is that it cannot be used to obtain an access token for an application.

Reply:
okay, will change the verbiage.